src/twisted/python/__init__.py

```diff
@@ -17,6 +17,13 @@
     "Please use constantly from PyPI instead.",
     "twisted.python", "constants")
 
+
+deprecatedModuleAttribute(
+    Version("Twisted", "NEXT", 0, 0),
```

Comment:
Just a side comment: is this NEXT thing documented somewhere?

I think that is important to have a docs/core/development/policy/compatibility-policy.rst talk about this.

And with this, maybe we can rename it to `~NEXT-RELEASE~` or something to make it clear that is a placeholder.

Also... why not  just `Version("Twisted", "NEXT")` why do we need the other zeros there?